modules/goldbachBidAdapter.js

```diff
@@ -0,0 +1,1168 @@
+import { Renderer } from '../src/Renderer.js';
+import * as utils from '../src/utils.js';
+import { config } from '../src/config.js';
+import { registerBidder, getIabSubCategory } from '../src/adapters/bidderFactory.js';
+import { BANNER, NATIVE, VIDEO, ADPOD } from '../src/mediaTypes.js';
+import { auctionManager } from '../src/auctionManager.js';
+import find from 'core-js-pure/features/array/find.js';
+import includes from 'core-js-pure/features/array/includes.js';
+import { OUTSTREAM, INSTREAM } from '../src/video.js';
+import { getStorageManager } from '../src/storageManager.js';
+
+const BIDDER_CODE = 'godlbach';
+const URL = 'https://ib.adnxs.com/ut/v3/prebid';
+const PRICING_URL = 'https://templates.da-services.ch/01_universal/burda_prebid/1.0/json/sizeCPMMapping.json';
+const URL_SIMPLE = 'https://ib.adnxs-simple.com/ut/v3/prebid';
+const VIDEO_TARGETING = ['id', 'minduration', 'maxduration',
+  'skippable', 'playback_method', 'frameworks', 'context', 'skipoffset'];
+const VIDEO_RTB_TARGETING = ['minduration', 'maxduration', 'skip', 'skipafter', 'playbackmethod', 'api'];
+const USER_PARAMS = ['age', 'externalUid', 'segments', 'gender', 'dnt', 'language'];
+const APP_DEVICE_PARAMS = ['geo', 'device_id']; // appid is collected separately
+const DEBUG_PARAMS = ['enabled', 'dongle', 'member_id', 'debug_timeout'];
+const DEFAULT_PRICE_MAPPING = {
+  '0x0': 2.5,
+  '300x600': 5,
+  '800x250': 6,
+  '350x600': 6
+};
+let PRICE_MAPPING;
+const VIDEO_MAPPING = {
+  playback_method: {
+    'unknown': 0,
+    'auto_play_sound_on': 1,
+    'auto_play_sound_off': 2,
+    'click_to_play': 3,
+    'mouse_over': 4,
+    'auto_play_sound_unknown': 5
+  },
+  context: {
+    'unknown': 0,
+    'pre_roll': 1,
+    'mid_roll': 2,
+    'post_roll': 3,
+    'outstream': 4,
+    'in-banner': 5
+  }
+};
+const NATIVE_MAPPING = {
+  body: 'description',
+  body2: 'desc2',
+  cta: 'ctatext',
+  image: {
+    serverName: 'main_image',
+    requiredParams: { required: true }
+  },
+  icon: {
+    serverName: 'icon',
+    requiredParams: { required: true }
+  },
+  sponsoredBy: 'sponsored_by',
+  privacyLink: 'privacy_link',
+  salePrice: 'saleprice',
+  displayUrl: 'displayurl'
+};
+const SOURCE = 'pbjs';
+const MAX_IMPS_PER_REQUEST = 15;
+const mappingFileUrl = 'https://acdn.adnxs-simple.com/prebid/appnexus-mapping/mappings.json';
+const SCRIPT_TAG_START = '<script';
+const VIEWABILITY_URL_START = /\/\/cdn\.adnxs\.com\/v|\/\/cdn\.adnxs\-simple\.com\/v/;
+const VIEWABILITY_FILE_NAME = 'trk.js';
+const GVLID = 32;
+const storage = getStorageManager(GVLID, BIDDER_CODE);
+
+export const spec = {
+  code: BIDDER_CODE,
```

Comment:
Just want to check with you if few things since appnexus gvlid is also 32:
- Is your publisher part of EEA. If not, we probably don't need this field.
- Do you have your own gvlid?

I'm not sure if I've seen two bid adapters with the same Gvlid, although there are aliases with same ids. I've to check with someone internally to see if we can allow same gvlid.

Reply:
removed :)